jboss-eap-cd-openshift-launch/added/standalone-openshift.xml

```diff
@@ -598,7 +598,6 @@
         <socket-binding name="ajp" interface="bindall" port="${jboss.ajp.port:8009}"/>
         <socket-binding name="http" interface="bindall" port="${jboss.http.port:8080}"/>
         <socket-binding name="https" interface="bindall" port="${jboss.https.port:8443}"/>
-        <socket-binding name="jgroups-mping" interface="private" multicast-address="${jboss.default.multicast.address:230.0.0.4}" multicast-port="45700"/>
```

Comment:
This is the CLOUD-1502 part. It made sense to test them all together.

Reply:
Wanted to ask about the very same change for `os-eap71-openshift/added/standalone-openshift.xml` but it's covered within https://github.com/jboss-openshift/cct_module/pull/281/commits/1fbbeee8aa554a791265bdea3f4a93bc0a7df75f, so it's OK.